scripts/minikube.sh

```diff
@@ -117,7 +117,7 @@ cephcsi)
     ;;
 k8s-sidecar)
     echo "copying the kubernetes sidecar images"
-    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-attacher:v1.2.0 "${K8S_IMAGE_REPO}"/csi-attacher:v1.2.0
+    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-attacher:v2.0.0 "${K8S_IMAGE_REPO}"/csi-attacher:v2.0.0
```

Comment:
(nit) we possibly also need to copy the 1.2.1 image to the cluster, as v1.13 kubernetes tests would use the same.

Reply:
Done and rebased